suite-native/module-send/src/components/CorrectNetworkMessageCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { CryptoIcon } from '@suite-native/icons';
+import { getNetworkType, networks, NetworkSymbol } from '@suite-common/wallet-config';
+import { Card, HStack, Text } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const cardStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundTertiaryDefaultOnElevation0,
+    borderColor: utils.colors.borderElevation0,
+    borderWidth: utils.borders.widths.small,
+    paddingVertical: utils.spacings.sp12,
+
+    ...utils.boxShadows.none,
+}));
+
+type CorrectNetworkMessageCardProps = {
+    networkSymbol: NetworkSymbol | null;
```

Comment:
networkSymbol can never be null when you show this card, can it?

Reply:
Correct 👍 54bcce4399